title-screen/title_screen.gd

```diff
@@ -0,0 +1,16 @@
+extends Control
```

Comment:
Could you move the files into different directories? They should roughly follow this structure:

- .gd -> scripts/
- .png -> sprites/
- .tscn -> scenes/

Feel free to make new directories under those to organise it better, e.g. a ui subdirectory